port_drayage_plugin/src/port_drayage_worker.cpp

```diff
@@ -102,6 +107,33 @@ namespace port_drayage_plugin
         return route_req;
     }
 
+    cav_msgs::UIInstructions PortDrayageWorker::compose_ui_instructions(const PortDrayageMobilityOperationMsg& msg, const std::string& previous_operation) {
```

Comment:
Looks like the only bit of the operations message that's required here is the operation type? Can we make this just accept that as a parameter instead of the full message?

Reply:
Fixed.